burst-gradle-plugin/src/test/projects/junit5/lib/src/test/kotlin/CoffeeTest.kt

```diff
@@ -0,0 +1,32 @@
+import app.cash.burst.Burst
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+
+@Burst
+class CoffeeTest(
+  private val espresso: Espresso,
+) {
+  @BeforeTest
+  fun setUp() {
+    println("set up $espresso")
+  }
+
+  @kotlin.test.Test
+  fun kotlinTestTest(dairy: Dairy) {
+    println("@kotlin.test.Test running $espresso $dairy")
+  }
+
+  @org.junit.Test
+  fun orgJunitTest(dairy: Dairy) {
+    println("@org.junit.Test running $espresso $dairy")
+  }
+
+  @org.junit.jupiter.api.Test
+  fun orgJunitJupiterApiTest(dairy: Dairy) {
+    println("@org.junit.jupiter.api.Test running $espresso $dairy")
+  }
+}
```

Comment:
Now it’s one test with three different test annotations. Hopefully nobody ever does this.